commons-http/src/test/scala/com/avsystem/commons/http/okhttp/rest/OkHttpHttpRestCallTest.scala

```diff
@@ -0,0 +1,31 @@
+package com.avsystem.commons
+package http.okhttp.rest
+
+import com.avsystem.commons.http.jetty.rest.{RestServlet, UsesHttpServer}
+import com.avsystem.commons.rest.RawRest.HandleRequest
+import com.avsystem.commons.rest.{AbstractRestCallTest, RestTestApi}
+import org.eclipse.jetty.server.Server
+import org.eclipse.jetty.servlet.{ServletHandler, ServletHolder}
+import org.scalatest.time.{Seconds, Span}
+
+import scala.concurrent.duration._
+import scala.language.postfixOps
+
+class OkHttpHttpRestCallTest extends AbstractRestCallTest
+  with UsesHttpServer
+  with UsesOkHttpClient {
+
+  override implicit def patienceConfig: PatienceConfig = PatienceConfig(Span(10, Seconds))
+
+  override protected def setupServer(server: Server): Unit = {
```

Comment:
Looks like this is copied from `JettyHttpRestCallTest`. Can we share it in base class or something?